src/core/server/ui_settings/create_or_upgrade_saved_config/create_or_upgrade_saved_config.ts

```diff
@@ -35,24 +35,28 @@ import { SavedObjectsErrorHelpers } from '../../saved_objects/';
 import { Logger } from '../../logging';
 
 import { getUpgradeableConfig } from './get_upgradeable_config';
+import { generateDocId } from '../ui_settings_client';
```

Comment:
Seems `create_or_upgrade_saved_config` and `ui_settings_client` import from each other, maybe move `generateDocId` to somewhere else?

Reply:
good catch, move it to utils